tests/units/nulecule/test_nulecule_component.py

```diff
@@ -293,6 +294,7 @@ def test_render_for_local_app_with_artifacts_for_provider(
             provider_key)
         self.assertEqual(nc.rendered_artifacts[provider_key],
                          expected_rendered_artifacts)
+    '''
```

Comment:
No problem as long as people in IRC and GH are aware of it, since test &&
install will fail on `make install`.

Let's wait :)